pkgs/os-specific/linux/open-iscsi/kmod-via-make.diff

```diff
@@ -0,0 +1,14 @@
+--- open-iscsi/usr/Makefile.orig	2016-04-24 13:51:26.301726559 +0200
```

Comment:
Looks to me like you could replace this patch with just setting the appropriate values in the package expression (see `NIX_CFLAGS_COMPILE` and `NIX_LDFLAGS`.